DeveloperTools/Controllers/RoutesController.cs

```diff
@@ -36,9 +37,9 @@ public ActionResult FindRoute(String url)
                 {
                     var ctx = new HttpContextWrapper(new HttpContext (new HttpRequest("", url, ""), new HttpResponse(new StringWriter(new StringBuilder()))));
                     
-                    if (r.Route is ContentRoute)
+                    if (r.Route is DefaultContentRoute)
```

Comment:
Should be able to use IContentRoute to avoid the use of internal namespaces(DefaultContentRoute) and cast r.Route to Route.